docs/src/deployment.md

```diff
@@ -3,6 +3,10 @@ hide:
   - navigation
 ---
 
+The default runtimes are available through separate repositories ([eoapi-cdk](https://github.com/developmentseed/eoapi-cdk) and [k8s-eoapi](https://github.com/developmentseed/k8s-eoapi)). 
+
+A demonstration application is accessible with the repository [eoapi-template](https://github.com/developmentseed/eoapi-template).
+
```

Comment:
I think we should really explain clearly what are the option for deploying an eoAPI like system. 

right now this page `deployment.md` is a mix of two things:
- links to external repo (eoapi-cdk and k8s-eoapi)
- a step-by-step `how to` to deploy eoapi custom demo application 

IMO this is really confusing. 

### Possible solution
1. create an `eoapi-docs` repo to host all the documentation about eoapi
2. use the current `eaopi` repo as an example of how eoapi can work

cc @sharkinsspatial @zacharyDez

Reply:
I just renamed the repo to `eoapi-k8s` so that will have to change too

Reply:
@vincentsarago We can also just remove the external references if you prefer, but I thought it would be helpful for someone interested in deployment to have this information readily available.

Reply:
@vincentsarago I agree this is confusing.  Ideally I think the eoAPI repo would have 3 deployment sections

1. How to deploy the eoAPI demonstration runtimes locally via docker.
2. A description of `eoapi-cdk` , `eoapi-cdk` and `eoapi-template` and pointers to their docs for deploying with the default runtimes they contain.
3. How to deploy the eoAPI demonstration runtimes using a local CDK app (mostly copy and pasted from `eoapi-template`)  that uses the `eoapi-cdk` constructs but with runtime overrides.

I think this is what @zacharyDez was shooting for with his PR but based on your feedback maybe we need a bit more clarification.

Reply:
@sharkinsspatial yep that's it on point!

Reply:
Got it! Will go for another iteration.